en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-310-to-320.md

```diff
@@ -1356,7 +1356,20 @@ these alerts are persisted individually as well in tables specific for the type
 data related to alerts by dropping this table.
 
 ??? info "DB Scripts"
+     ```sql
      DROP TABLE APIMALLALERT;
+     ```
+
+!!! note
+    Only the `ANALYTICS_DB` need to be migrated and the other DBs (e.g. `DASHBOARD_DB`) can either be pointed to a new DB or be cleaned. If there are custom dashboards, those dashboards need to be exported and imported accordingly.
+
+    To export a dashboard, you need to login as an admin to dashboard listing page and select **Export** from the drop down menu (3 vertical dots) next to the dashboard name.
+
+    <img src="{{base_path}}/assets/img/setup-and-install/analytics_dashboard_export.png" alt="Dashboard_export" width="750"/>
+
+    This will download the relevant `<dashboard-name>.json` file. To import the dashboard, copy the generated JSON file to `<API-M_ANALYTICS_HOME>/wso2/dashboard/resources/dashboard`. 
+
+    For more information on importing dashboards, see [Importing Dashboards](https://docs.wso2.com/display/SP430/Importing+and+Exporting+Dashboards#ImportingandExportingDashboards-Importingdashboards).
```

Comment:
Do we need to point to this doc? I think we can exclude this reference since we have already provided the directory path to copy the dashboard json file and this link wouldn't provide anymore details on it.

Reply:
Fixed with 8144662